pkgs/by-name/cl/clash-verge-rev/package.nix

```diff
@@ -68,10 +82,7 @@ let
       Guanran928
       bot-wxt1221
     ];
-    platforms = [
-      "x86_64-linux"
-      "aarch64-linux"
-    ];
+    platforms = lib.platforms.linux;
```

Comment:
We probably want to keep the old explicit platforms as linux contains also i686 and things like powerpc and whatnot.